### DISCUSSION TITLE
My copilot was limited, How can I unblock it

### DISCUSSION THREAD

**Discussion by Tianci-King** [community/community#122292] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

<img width="471" alt="iShot_2024-05-04_21 06 38" src="https://github.com/community/community/assets/109196852/db815793-0898-4b37-8457-ec2dae6251d0">

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
